public/app/features/explore/Logs.tsx

```diff
@@ -305,6 +320,9 @@ export class UnthemedLogs extends PureComponent<Props, State> {
             <InlineField label="Wrap lines" transparent>
               <InlineSwitch value={wrapLogMessage} onChange={this.onChangewrapLogMessage} transparent />
             </InlineField>
+            <InlineField label="Restructure" transparent>
```

Comment:
We should decide what is the naming of this going to be going to be and keep it consistent, cause somewhere you are using prettify and somewhere restructure and ideally the naming would be same. Maybe we can do quick check on standup? 🙂

Reply:
Yes i agree, that's a very good idea! Let's get everyone's opinion on the standup today 😄